src/test/java/org/openstreetmap/atlas/tags/filters/TaggableFilterTest.java

```diff
@@ -57,10 +82,10 @@ public void testParsing()
     @Test
     public void testSerialization() throws ClassNotFoundException
     {
-        final String definition = "amenity->bus_station|highway->BUS_STOP|bus->*^trolleybus->*&public_transport->stop_position,platform,station";
-        final TaggableFilter filter = new TaggableFilter(definition);
+        final String definition = "amenity->bus_station|highway->BUS_STOP|bus->*||trolleybus->*&public_transport->stop_position,platform,station";
```

Comment:
Looking at the README, backwards compatibility means that the ^ notation should still work, correct? Is there a reason the unit test was changed?

You can tell I don't want to refactor the `AtlasMissingShardVerifier` :)

Reply:
I modified the test to be using the new syntax, however, I added a new test to verify that the backwards compatibility was not dropped: https://github.com/osmlab/atlas/pull/229/files/fd9e83b08b95dd1ca07133fc431f6f2284378765#diff-66d20c24969469313d66a1c180052f0cR20

Reply:
Thanks, not sure how I missed that!